pydevd_file_utils.py

```diff
@@ -231,8 +232,28 @@ def _NormPaths(filename):  #Let's redefine _NormPaths to work with paths that ma
     #Don't fail if there's something not correct here -- but at least print it to the user so that we can correct that
     traceback.print_exc()
 
-
-if PATHS_FROM_ECLIPSE_TO_PYTHON:
+norm_file_to_client = None
+norm_file_to_server = None
+
+def setup_client_server_paths(paths):
+    '''paths is the same format as PATHS_FROM_ECLIPSE_TO_PYTHON'''
+    
+    global NORM_FILENAME_TO_SERVER_CONTAINER
+    global NORM_FILENAME_TO_CLIENT_CONTAINER
+    global PATHS_FROM_ECLIPSE_TO_PYTHON
+    global norm_file_to_client
+    global norm_file_to_server
+    
+    NORM_FILENAME_TO_SERVER_CONTAINER = {}
+    NORM_FILENAME_TO_CLIENT_CONTAINER = {}
+    PATHS_FROM_ECLIPSE_TO_PYTHON = copy.deepcopy(paths)
```

Comment:
Well, deepcopy gains nothing in this case (since the contents of the list should be immutable doing a shallow copy with paths[:] or a deepcopy should give the same results) -- so, in this case there are only cons of using deepcopy (i.e.: it's slower and duplicates memory).